girder/cli/build.py

```diff
@@ -63,6 +63,19 @@ def main(dev, mode, watch, watch_plugin, npm, reinstall):
         npmLockFile = os.path.join(staging, 'package-lock.json')
         if os.path.exists(npmLockFile):
             os.unlink(npmLockFile)
+
+        # Remove any lingering node_modules to ensure clean install
+        plugin_dirs = [path.replace('file:', '') for path in _collectPluginDependencies().values()]
+        dirs = [staging, os.path.join(staging, 'src')]
+        dirs.extend(plugin_dirs)
```

Comment:
FWIW if you want a more functional version, you can use syntactic sugar:

```python
dirs = [staging, os.path.join(staging, 'src')] + plugin_dirs
```

Reply:
Thanks, forgot about that.